design/how-ironic-works.md

```diff
@@ -40,56 +40,40 @@ have to become an expert in Ironic in order to learn to leverage it.
 ## How ironic controls hardware
 
 Ironic is largely designed around the ability to issue commands to a
-remote Baseboard Management Controller (BMC) in order to assert the
-desired next boot device and control system power state.
+remote Baseboard Management Controller (BMC) in order to control the
+desired next boot device and the system power state.
 
-BMCs are also not required, however lacking BMC functionality, then
-power and potential boot mode changes need to be performed by an
-external entity such as a human or a network attached power distribution
-unit.
-
-- Access to the BMC can be via a routed IP network,
-  however this may be less desirable from a security standpoint.
-
-- Some vendor driver interfaces may expect to assert configuration
-  to a BMC where the BMC is expected to reach controller resources
-  or hosted services.
-  Mainly this should be expected when virtual media is in use, which means
-  some sort of dedicated BMC network access becomes desirable, as long as
-  the connectivity matches the failure domain of the machines upon which
-  ironic will be executing.
+Without BMC functionality, the power and potential boot mode changes
+need to be performed by an external entity such as a human or a
+network attached power distribution unit.
 
 ## How ironic boots hardware
 
-Typically harware is booted utilizing PXE. In the most ideal scenario this
+Typically nodes are booted utilizing PXE. In the most ideal scenario this
 would be a hybrid PXE/iPXE configuration such that the deployment ramdisk
 for ironic is able to be quickly and efficently transferred to the
-baremetal machine.
+node.
 
 In order to assert this configuration at boot time, a dedicated DHCP server
-on semi-dedicated "provisioning" network should be leveraged. This network
+on a semi-dedicated "provisioning" network should be leveraged. This network
 may be reused by other services and systems, but can also be leveraged in
 the case of hardware discovery. The most important factor is that this
-network does not have an external DHCP server attached.
+network does not have a second DHCP server attached.
 
 Some specific hardware types in ironic do support use of virtual media
 to boot the deployment ramdisk, however this functionality is not
-available with vendor neutral IPMI. Newer protocols such as Redfish
+available with the vendor neutral IPMI driver. Newer protocols such as Redfish
 may support virtual media, but not as of the time of this document
-having been composed. Use of virtual media should be considered
-useful, but not well enough supported at this time to be considered useful.
+having been composed. Virtual media should be considered
+not well enough supported at this time to be considered useful.
 Other methods such as booting directly from iSCSI should be considered
-out-of-scope in this use case as they are centered around an external block
+out-of-scope in this use case as they are require an external block
```

Comment:
Actually, you can tell ironic to make static configurations to a known iscsi san..... but require works.

Reply:
I was trying to emphasize that it needs something that won't be provided by MetalKube or a typical kubernetes cluster.